cypresse2e/cypress/elements/pages/post.js

```diff
@@ -0,0 +1,94 @@
+class Post{
+
+    likeButton = 'div[id^="HB_LK"]';
+    commentButton = 'div[id^="HB_CM"]';
+    shareButton = 'div[id^="PO_SH"]';
+    postHeader = 'div.card-header';
+    postTitle = 'h4';
+    postAuthorUrl = 'div.card-header > a';
+    modalWindowShareVia = 'div[role="document"]';
+    modalWindowShareViaH4Title = 'div.am-modal-body h4';
+    modalWindowCloseButton = 'span.am-modal-close-x';
+    modalWindowEmailButton = 'button[aria-label="email"]';
+    modalWindowFacebookButton = 'button[aria-label="facebook"]';
+    modalWindowLinkedinButton = 'button[aria-label="linkedin"]';
+    modalWindowRedditButton = 'button[aria-label="reddit"]';
+    modalWindowTelegramButton = 'button[aria-label="telegram"]';
+    modalWindowTwitterButton = 'button[aria-label="twitter"]';
+    modalWindowWhatsappButton = 'button[aria-label="whatsapp"]';
+    modalWindowSharingUrlInput = 'div > input[type=text]';
+
+
+    constructor() { }
+
+    getLikeButtonSelector(){
+        return this.likeButton;
+    }
+
+    getCommentButtonSelector(){
+        return this.commentButton;
+    }
+
+    getShareButton(){
+        return cy.get(this.shareButton);
+    }
+
+    getShareButton(){
```

Comment:
Hi @galeza! I see here a duplication. for getShareButton()

Reply:
Removed